lib/tldr-lint.js

```diff
@@ -200,5 +201,9 @@ linter.processFile = function(file, verbose, alsoFormat) {
     result.errors.push({ locinfo: { first_line: '0' }, code: 'TLDR108', description: this.ERRORS.TLDR108 });
   }
 
+  if (RegExp(/[A-Z]/).test(path.basename(file))) {
```

Comment:
I think it's clearer to just have this be a proper regex literal (which is what it should be as it's a constant), and not rely on the weird factory schemantics of builtin constructors:
```suggestion
  if (/[A-Z]/.test(path.basename(file))) {
```

Reply:
D'oh. Completely forgot it's possible to do that in JS. 👍🏻